packages/design-system/src/styles/typography-body-text.stories.tsx

```diff
@@ -28,36 +28,26 @@ export const AllBodyText = () => {
     </>
   );
 };
+
 export const AllBodyTextOnDark = AllBodyText.bind({});
```

Comment:
These changes aren't related. Can this and the other typography stories be removed from the diff?

Reply:
Earlier I mentioned the calendar picker drop shadow.